pkg/updater/service/helper/main.go

```diff
@@ -0,0 +1,123 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package main is a package that allows dd-updater
+// to execute a subset of priviledged commands
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"os/user"
+	"strconv"
+	"strings"
+	"syscall"
+	"unicode"
+)
+
+var (
+	installPath string
+	systemdPath = "/lib/systemd/system" // todo load it at build time from omnibus
```

Comment:
still figuring this one out, if both /lib/systemd/system and /usr/lib/systemd/system/ are not present, should I add /usr/lib/systemd/system/?

Reply:
IMO it's is sensible to create it. Else, we could run `systemd-analyze --global unit-paths` at setup and take the first entry that exists from the bottom up (that should be the one with the lowest priority). WDYT?